app/src/test/java/com/jetbrains/handson/mpp/mobile/ApplicationPresenterTest.kt

```diff
@@ -0,0 +1,61 @@
+package com.jetbrains.handson.mpp.mobile
```

Comment:
It might be worth looking into looking into how to put the tests into the `CommonTest` since these tests are platform specific to android even though they're testing the core functionality.